code_files/main.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+from file_parser import Parser
```

Comment:
`Parser` --> `WeatherParser`

Reply:
[Resolved in this commit](https://github.com/ttqureshi/WeatherMan/commit/a9bfba76030779f62875a46887178ea2565ca521)